tutorials/kube-mqtt/refresher-container/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cryptography==2.6.1
```

Comment:
```
cryptography==2.7
```

and

```
psutil==5.6.3
```

Reply:
so now I've just discovered https://pypi.org/project/pur/